lib/phctl.rb

```diff
@@ -179,6 +179,13 @@ def shared_print_newly_ingested
       run_common_job(Reports::SharedPrintNewlyIngested, options)
     end
 
+    desc "shared-print-phase-count (--phase=x)",
+      "Get tally of commitments per organization in the given phase"
+    option :phase, type: :numeric, default: nil
+    def shared_print_phase_count
+      run_common_job(Reports::SharedPrintPhaseCount, options)
```

Comment:
Should add this to `phctl_spec.rb` & consider adding something to `integration_spec.rb`

Reply:
Yep and yep.